cli/list.go

```diff
@@ -115,10 +127,21 @@ func list() *cobra.Command {
 			return err
 		},
 	}
+
+	v := reflect.Indirect(reflect.ValueOf(displayWorkspaces))
+	availColumns, err := cliui.TypeToTableHeaders(v.Type().Elem())
+	if err != nil {
+		panic(err)
+	}
+	for i, s := range availColumns {
+		availColumns[i] = strings.Replace(s, " ", "_", -1)
+	}
+	columnString := strings.Join(availColumns[:], ", ")
```

Comment:
This is good, but since we'd probably want to use this in multiple commands it would be better served as a function in the cliui package that accepts `any`:

```go
func TableHeaders(t any) ([]string, error) {
	v := reflect.Indirect(reflect.ValueOf(t))
	return cliui.TypeToTableHeaders(v.Type())
}

func FormatColumnNames(columns []string) string {
	out := strings.Join(columns[:], ", ")
	out = strings.ReplaceAll(out, " ", "_")
	return out
}
```

which could be called like

```go
columns, err := cliui.TableHeaders(workspaceListRow{})
if err != nil {
	panic(err)
}

columnString := cliui.FormatColumnNames(columns)
```

This would also remove the need for having a global(ish) var for the output

Reply:
future fix ideaaa :D